src/components/Counter/Counter.css

```diff
@@ -0,0 +1,14 @@
+@value breakpoint, fontstack from 'styles/variables.css';
+@value black from theme(colors);
+
+.root {
+  color: black;
```

Comment:
`black` is the variable name imported above. it's just a proof of concept but I guess we should go with something like `colorBlack` for clarity